### DISCUSSION TITLE
Assign permissions to tenant

### DISCUSSION THREAD

**Discussion by StandaChromec** [netbox-community/netbox#14107] [Q&A]
Hi,

I'm looking for a way to grant permissions to object (e.g. device, IP address, prefix ...) to its tenant. Tenants are synchronized via API from Active Directory. Every tenant do have email attribute which is unique. We do have LDAP module enabled. So users are authenticated with their LDAP (Active Directory) credentials.

Is there a way to create some link between LDAP user and tenant by email address and define permission allowing to modify object by its tenant?

Thanks a lot!

Standa

**Comment by dreng** [comment]
Hi,

NetBox rights don't work this way. You can only grant access to models (aka tables). It is not possible to grant access only to certain data sets of a model.
